scripts/paths.js

```diff
@@ -1,22 +1,26 @@
-module.exports = {
-  "/thank-you": {
-    name: "thank-you",
-    path: "/thank-you/?",
-    handler: require('../pages/thank-you.jsx')
-  },
-  "/": {
-    name: "sequential",
-    path: "/?",
-    handler: require('../pages/sequential.jsx')
-  },
-  "/share": {
-    name: "share",
-    path: "/share/?",
-    handler: require('../pages/share.jsx')
-  },
-  "/give-bitcoin": {
-    name: "give-bitcoin",
-    path: "/give-bitcoin/?",
-    handler: require('../pages/give-bitcoin.jsx')
-  }
-};
+import currencies from './currencies.js';
+import locales from "../locales";
+import pages from "./pages.js";
+
+function getLocalePaths(arr) {
```

Comment:
Lets optimize this one a bit.

 ``` js
function getLocalePaths(paths, locales) {
  var returnArray = [];
  paths.map(function(path) {
    returnArray = returnArray.concat(locales.map(function(locale) {
      return path + locale;
    }));
  });
  return returnArray;
}
```

This way we can just call it like this:

``` js
getLocalePaths(['/sequential-', '/thank-you-'], ['en', 'th'])
// return Array [ "/sequential-en", "/sequential-th", "/thank-you-en", "/thank-you-th" ]
```

You can also take this further by merging `locales` array with `currencies` array and just call this function once like this

``` js
getPaths((['/sequential-', '/thank-you-', '/paypal-donate-'], ['en', 'th', 'USD', 'CAD']);
```

I don't know what you think about this approach?